src/components/Social.js

```diff
@@ -19,12 +19,14 @@ export default (editor, { dc, coreMjmlModel, coreMjmlView }) => {
           'border-radius', 'border-top-left-radius', 'border-top-right-radius', 'border-bottom-left-radius', 'border-bottom-right-radius',
           'container-background-color',
           'color',
+          'vertical-align'
```

Comment:
This should only be in SocialElement instead of Social, right?

Reply:
you're right, I'll fix it !